src/Test.UnitTests.Sarif.Multitool.Library/ApplyPolicyCommandTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Moq;
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class ApplyPolicyCommandTests
+    {
+        private static readonly ResourceExtractor Extractor = new ResourceExtractor(typeof(ApplyPolicyCommandTests));
+
+        [Fact]
+        public void WhenOutputFormatOptionsAreInconsistent_Fails()
+        {
+            const string InputFilePath = "AnyFile.sarif";
+
+            var mockFileSystem = new Mock<IFileSystem>();
+            mockFileSystem.Setup(_ => _.FileExists(InputFilePath)).Returns(true);
+
+            var options = new ApplyPolicyOptions
+            {
+                InputFilePath = InputFilePath,
+                Inline = true,
+                PrettyPrint = true,
+                Minify = true
```

Comment:
There are tests elsewhere that verify the consistent treatment of `--minify` and `--pretty-print`, so we don't need this test.